app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageComposerState.kt

```diff
@@ -83,6 +86,9 @@ data class MessageComposerState(
 ) {
     var messageComposeInputState: MessageComposeInputState by mutableStateOf(MessageComposeInputState.Inactive())
         private set
+
+    private var currentSelfDeletingStatus: SelfDeletingMessagesStatus by mutableStateOf(SelfDeletingMessagesStatus(true, null, ZERO))
```

Comment:
Should we start state with selfDeleting enabled? I think default behaviour is that it is disabled? 🤔

Reply:
That would mean that if for any reason the first sync fails (or consult to the DB fails), users will not be able to use that feature, so tbh I'm not entirely sure we should disable it per default.